.github/workflows/test-rector.yml

```diff
@@ -45,7 +45,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        php-versions: ['7.4', '8.0']
+        php-versions: ['8.0', '8.2']
```

Comment:
```suggestion
        php-versions: ['8.0', '8.1', '8.2']
```

Need test rector for `8.1`?

Reply:
Why need?
We want as few heavy checks as possible.

Reply:
Lowest and highest should be sufficient.